### PR TITLE
[RTM] Fix the search rebuilding cors headers

### PR DESCRIPTION
**Pull request by Toflar**

### CODE REVIEW

File: src/Resources/config/cors.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+nelmio_cors:
+    paths:
+        '/': # Nonsense configuration as the nelmio compiler pass will abort if not present
```
Comment (leofeyer) [inline src/Resources/config/cors.yml]: Why don't we use `'^/'` as in the [example configuration](https://github.com/nelmio/NelmioCorsBundle#configuration)?

Comment (Toflar) [inline src/Resources/config/cors.yml]: It's completely irrelevant. I'd rather not use anything at all.

**Comment by leofeyer** [comment]
I would prefer to have the plugin related changes in a separate PR (there is contao/manager-bundle#14 already).

**Comment by Toflar** [comment]
Sorry but I don't find the energy to split it up again. The other PR is closed for a reason.

**Comment by leofeyer** [comment]
You told me not to mix multiple features in one pull request. 😄

**Comment by leofeyer** [comment]
But I can split the PR myself, of course.

**Comment by Toflar** [comment]
Yes, which is not the case here. Absolutely all changes are related to the very same bug fix.

**Comment by leofeyer** [comment]
Not really, because we are not yet storing the manager configuration in the core bundle. This is a feature which is yet to be merged and not to be introduces half-way in a PR.

**Comment by Toflar** [comment]
I disagree. But I don't have the time to discuss this, feel free to split it into multiple PR's.

**Comment by aschempp** [comment]
Not true, we already do for security config.

**Comment by leofeyer** [comment]
@aschempp Which is wrong and has been changed in the `develop` branch.

File: src/Resources/config/cors.yml (hunk 1 above)
Comment (leofeyer) [inline src/Resources/config/cors.yml]: I'd go for `'^/'` as we are applying the logic to all routes, aren't we?

Comment (leofeyer) [inline src/Resources/config/cors.yml]: I have created a PR to fix this: https://github.com/nelmio/NelmioCorsBundle/pull/73

**Comment by aschempp** [comment]
why was this merged in develop instead of the hotfix branch?

**Comment by leofeyer** [comment]
Originally as basis for the other PR, which then has been closed. It has a "needs backport" label though.

**Comment by leofeyer** [comment]
Back ported in 002a7a302c6b97d877f8004a30f0c7dba6693df7.

**Pull request merged**